bazel/linux/qemu.bzl

```diff
@@ -0,0 +1,121 @@
+load("//bazel/linux:runner.bzl", "CREATE_RUNNER_ATTRS", "create_runner")
+load("@bazel_skylib//lib:shell.bzl", "shell")
+
+DEFAULT_QEMU_FLAGS = [
+    "-enable-kvm",
+    "-cpu",
+    "host",
+    "-machine",
+    "pc,accel=kvm,usb=off,dump-guest-core=off",
```

Comment:
Minor but good to know that this will fail on presubmit. There is no KVM there. I was doing `accel=kvm:tcg`.

Reply:
if that's ok to you, will change the default in a follow up PR as I see this failing, or add some shell hackery to pick the best one depending one environment.